README.md

```diff
@@ -214,30 +380,11 @@ There are two use cases for deep linking that can be relevant here:
 
 In order for deep linking to work, there are a few configurations that are needed and these are no different from what are required for handling deep linking in general and [Apple documentation](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app) on this can be followed in conjunction with the steps highlighted here:
 
-### Option 1: Modify Open Tracking
-
-If you plan to use universal links in your app for deep linking you will need to modify the push open tracking as described below:
-
-```swift
-extension AppDelegate: UNUserNotificationCenterDelegate {
-    func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
-        let handled = KlaviyoSDK().handle(notificationResponse: response, withCompletionHandler: completionHandler) { url in
-            print("deep link is ", url)
-        }
-        if !handled {
-           // not a klaviyo notification should be handled by other app code
-        }
-    }
-}
-```
-
-Note that the deep link handler will be called back on the main thread. If you want to handle uri schemes in addition to universal links you implement them as described below.
-
-### Option 2: Use URL Schemes
+##### Option 1: URI Schemes
 
 If you do not need universal link support you can instead implement url schemes for your app and the deepLinkHandler as indicated in Option 1 can be omitted. The Klaviyo SDK will follow all url automatically in this case.
```

Comment:
I think we can remove this line since it references the old order of the readme.

Reply:
Instead maybe we should say something like "URL schemes are the traditional and simpler way of deep linking from a push notification to your app. However, these links will only work if your mobile app is installed on a device and will not be understood by a web browser if, for example, you want to link from an email to your app."

Reply:
Hmm, so yeah I missed updating this but I think this is calling out that if your code is not going to use universal links that you don't need to implement some code related to universal links. But now that we switched the ordering, we can remove that and add context with what you suggested here.